shell/driver.greenplum/db-query

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# db-query -- Evaluates given SQL query against the PostgreSQL database configured for a DeepDive application
+# > eval "$(db-parse "$url")"
+# > db-query SQL FORMAT HEADER
+##
+set -eu
+
+sql=${1:?No SQL given}; shift
+format=${1:?No FORMAT given}; shift
+header=${1:?No HEADER given}; shift
+
+copy_option=
+case $header in
+    0) ;;
+    1) copy_option+=" HEADER" ;;
+    *) error "$header: unrecognized value for HEADER"
+esac
+case $format in
+    tsv) ;;
+    csv) copy_option+=" csv" ;;
+    json)
+        db-query "$sql" csv 1 | python $DEEPDIVE_HOME/util/csvtojson.py "$sql"
```

Comment:
You don't need to call the python script with full path as long as it stays in either util/ or bin/.  I don't think explicitly calling python is a good idea, either.  So, this can be written simply as:
```bash
db-query "$sql" csv 1 | csvtojson.py "$sql"
```

However, since we want to call the magic sql directly from this script and pass the types to python, it should be something like this:
```bash
types=$(db-prompt -c "CREATE TEMP TABLE ... ($sql LIMIT 0); ... magic SQL ...")
db-query "$sql" csv 1 | csvtojson.py $types  # XXX assuming no spaces in type names
```